src/utils/log.js

```diff
@@ -1,6 +1,12 @@
-/* eslint-disable no-console */
-/* global console, window, Image */
+import {Log} from 'probe.gl';
+const log = new Log({id: 'luma'});
+
+export default log;
 
+// TODO - delete when confident that probe.gl logging implements all opts
```

Comment:
Should we still make old `log` importable (as log_old or something similar), until we completely remove it, for the sake of comparing if find any issues with probe.gl version.

Reply:
> Should we still make old log importable 

Yes, why not. I uncommented it and exported as `oldLog`